src/views/virtualmachines/details/tabs/configuration/scheduling/components/SchedulingSection/SchedulingSection.tsx

```diff
@@ -50,12 +49,7 @@ const SchedulingSection: React.FC<SchedulingSectionProps> = ({ vm, pathname }) =
         <LinkIcon size="sm" />
       </a>
       <Title headingLevel="h2" className="co-section-heading">
-        <Flex>
-          <FlexItem>{t('Scheduling and resources requirements')}</FlexItem>
-          <FlexItem>
-            <SidebarEditorSwitch />
-          </FlexItem>
-        </Flex>
+        {t('Scheduling and resources requirements')}
```

Comment:
I believe this should be 'resource' instead of 'resources'.

Reply:
It was always "resources". See the original line of the code just above this change. Also see the VM Template's Scheduling tab. Thanks.

Reply:
I believe he meant that it's not right, even if it used to be like that

Reply:
Aviv is correct. In this case, making 'resource' plural isn't proper English.

Reply:
It's "resource requirements" in this context because "resource" is an adjective, modifying the noun "requirements". Adjectives do not have singular or plural forms. 

Examples: blue rooms, wide streets, restaurant receipts, lunch bags, workplace policies, human resources.

Reply:
Thank you very much, Avital!